package.json

```diff
@@ -288,7 +294,7 @@
     "tsconfig-paths-webpack-plugin": "3.5.2",
     "tslib": "2.4.0",
     "typescript": "4.3.5",
-    "webpack": "5.44.0",
+    "webpack": "5.72.1",
```

Comment:
bump needed so webpack 5 is deduped ( typings package has webpack as dependency )